lib/app/modules/languages/storage_service.dart

```diff
@@ -0,0 +1,20 @@
+
+import 'package:get/get.dart';
+import 'package:get_storage/get_storage.dart';
+
+class StorageService extends GetxService{
```

Comment:
We needed to eventually create a provider for Get Storage since a couple of new features rely on persistent storage and secure_storage isn't exactly meant for that. Please create a new provider called get_storage_provider. We can migrate some functions from secure_storage to this later on.